_posts/2020-08-24-gsoc-report-cbouy.md

```diff
@@ -0,0 +1,70 @@
+  This PR allows us to draw images (SVG, PNG, and GIF) of AtomGroups using RDKit. It also adds rich displays to AtomGroups in notebooks (a.k.a. `__repr__` methods).
+  Before working on this, I thought the only way to use alternative representations for Python objects was to define a `_repr_*_` method for your class, where * is a MIME type such as png, html...*etc.* There is actually a second way, where you tell IPython directly how it's supposed to represent an object. This allows funky representations of any object, even python built-in types, *i.e.* `int` as roman numerals and so on. I also wrote my first metaclass here, to register different "viewer" classes when more become available in the future. It's also not straightforward to write tests for the images as different versions of RDKit or other packages will lead to slightly different outputs.
+  More discussion, code review, and tests are needed before this is ready.
+
+* **Wrap RDKit descriptors and fingerprints**: [#2912](https://github.com/MDAnalysis/mdanalysis/pull/2912/files/76a51ece7b676a5f5ba3d683fdf942706f87c141)  
+  This PR adds new kinds of analysis that are typically performed on small molecules in the chemoinformatics field. Fingerprints are mostly used for calculating similarity metrics between molecules and a reference. Molecular descriptors could be used to describe all the sampled conformations of a ligand in a binding pocket during a simulation, and given as input to a machine-learning model for clustering, scoring binding poses...*etc.*  
+  This is currently missing more discussion, documentation, and a few tests.
+
+* **Change the Converters API**: [#2882](https://github.com/MDAnalysis/mdanalysis/pull/2882/files/43f6b34a3f7cfe57ed5fa296dbc800f304e95198)   
+  While developing the RDKitConverter, some interesting points were made about the current API used to convert AtomGroups, *i.e.* `u.atoms.convert_to("RDKIT")`. This method is case sensitive, it doesn't allow to pass arguments to the converter class, and it requires users to read the documentation to know which converters are available. This PR corrects the two first points and adds the possibility to either use the previous syntax or tab-completion to find the available converters *i.e.* `u.atoms.convert_to.rdkit()`.  
+  This was inspired by pandas `df.plot(kind="scatter", ...)` which is also accessible as `df.plot.scatter(...)`.  
+  Since this is an API change, more discussion with core developers is needed for now.
+
+### Left to do
+
+* Guessers for aromaticity and Gasteiger charges through RDKit
+* Tutorial on the reader/converter and wrapped RDKit functionalities in the UserGuide. This will make it easier for users to know about the features I implemented and how to use them properly, rather than searching for every single feature in the documentation.
+* Documentation on the RDKit format in the UserGuide: [#69](https://github.com/MDAnalysis/UserGuide/pull/69)
+
```

Comment:
Can you include a section here that demoes some of the capabilities? You had great blog posts doing that. Show some images here. Give readers a taste of what they can get.